### PR TITLE
Bhagyashree_Assignment10

### PR DESCRIPTION
**Pull request by BhagyashreeJYeole**
image

### CODE REVIEW

File: src/bhagyashree/Assignment10.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package bhagyashree;
+
```
Comment (MadhaviKhasbage) [inline src/bhagyashree/Assignment10.java:2]: Add assignment text here, so you can understand the requirement after 2-3 month also

File: src/bhagyashree/Assignment10.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	void isPrimeNumber(int number) {
+		boolean status = true;
+		for (int index = 2; index < number; index++) {
+			if (number % index == 0) {
+				status = false;
+				System.out.println(number + " is not prime");
+				break;
+			}
+		}
```
Comment (MadhaviKhasbage) [inline src/bhagyashree/Assignment10.java:16]: Program out put is wrong,
you should have written below logic in open and close curly bracket. your logic get changed coz you did not add brackets. Add brackets and check.

if (status == true){
		System.out.println(number + " is prime");
		count++;
		sum = sum + number;
		average = sum / number;
}

**Comment by BhagyashreeJYeole** [comment]
Done with changes
image

**Review by MadhaviKhasbage — APPROVED** [review]
now code looks good to me!!

**Pull request merged**